src/app/components/wrapper-components/part-creation/part-creation.component.ts

```diff
@@ -512,6 +510,13 @@ export class PartCreationComponent implements OnInit, OnDestroy {
         this.candidatesSubscription.unsubscribe();
 
         this.stopSendingPresenceTokensAndObservingUsersIfNeeded();
+        if (this.connectedUserService.user.isAbsent()) {
+            // User disconnected, there's not much we can do at this point
+            // We could instead remove parts in creation when doing the log out,
+            // but this is an unlikely event and just ignoring log outs here
+            // treats this similar to a "tab closed" event, so it is more consistent behavior.
+            return;
+        }
```

Comment:
ça sert à quoi ?

Reply:
t'es déconnecté, tu peux plus changer ton utilisateur.
On veut pas essayer de modifier la DB quand on est pas autorisé

Reply:
vrai! donc là, les déconnection depuis un autre tab font ça ?
C'est sûr que tout ce qui est dans ngOnDestroy c'est soit changement de page, et là c'est fait, mais c'est ce dans un changement de page c'est fait.

Reply:
bonne question, à tester la déconnexion depuis un autre tab, m'est avis que c'est pas quelque chose qu'on a tellement testé jusqu'ici !

Reply:
bonne question, à tester la déconnexion depuis un autre tab, m'est avis que c'est pas quelque chose qu'on a tellement testé jusqu'ici !

Reply:
ah mais somme toute içi avant, soit ça faisait ce que ça faisait, soit ça throwais, maintenant on s'assure juste que ça throwais.

Cependant la seule chance qu'ici, ça soit une branche utilisable live, c'est une déconnection depuis un autre tab, vérifierais ton plutôt que de rajouter du test sur un comportement qu'on a pas testé en live ?

Reply:
hmm maintenant ça throw pas. Je réitère : à tester en live avant tout. C'est pas une question de multi-tab me semble, c'est juste : tu crées une partie et puis tu te déconnecte.

Reply:
Ah oui.
On a un ticket (de bug potentiel) pour le cas : 
"je suis en train de créer une partie, je crée un nouvel onglet, ou je clique 'déconnection' je suis toujours créateur de la partie ou je suis plus présent ?"
Si non tu peux créer ?

Reply:
créé